.editorconfig

```diff
@@ -8,12 +8,19 @@ end_of_line = lf
 charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
+block_comment_start = /**
+block_comment = *
+block_comment_end = */
```

Comment:
Could you remove these, please?

Reply:
Removing right away 🏃🏻‍♂️ 
(the checker tool `eclint` needs to know about block comment signs)

Reply:
Removed.